src/lib/expansion/accordion-item.ts

```diff
@@ -0,0 +1,27 @@
+import {Output, EventEmitter, OnDestroy} from '@angular/core';
+
+/** Used to generate unique ID for each expansion panel. */
+let nextId = 0;
+
+
+/**
+ * An abstract class to be extended and decorated as a component.  Sets up all
+ * events and attributes needed to be managed by a CdkAccordion parent.
+ */
+export abstract class CdkAccordionItem implements OnDestroy {
```

Comment:
Can definitely do this, but should we keep the `Cdk` prefix since it should be used as the super for a Directive/Component.